source/extensions/filters/network/well_known_names.h

```diff
@@ -47,6 +47,8 @@ class NetworkFilterNameValues {
   const std::string ExtAuthorization = "envoy.filters.network.ext_authz";
   // Kafka Broker filter
   const std::string KafkaBroker = "envoy.filters.network.kafka_broker";
+  // Kafka Mesh filter
+  const std::string KafkaMesh = "envoy.filters.network.kafka_mesh";
```

Comment:
we are trying to get rid of this file. Can you just inline this?

Reply:
done